extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -224,9 +224,17 @@ export class AmpAnalytics extends AMP.BaseElement {
         this.instrumentation_.createAnalyticsGroup(this.element);
 
     if (this.config_['transport'] && this.config_['transport']['iframe']) {
+      let ampAdResourceId;
+      try {
+        ampAdResourceId = this.element.ownerDocument.defaultView
```

Comment:
Can we create a helper function here? Please refer to `getParentWindowFrameElement(`

Reply:
Done